app/assets/javascripts/moderation/ban.coffee

```diff
@@ -1,22 +1,27 @@
 load = ->
-  parent = $ "#ban-control-super"
-  return unless parent.length > 0
+  return unless document.getElementById('ban-control-super') != null
+  modalEl = $("#modal-ban")
+  modalEl.modal "hide"
+  modalForm = modalEl.find("form")[0]
+  banCheckbox = modalForm.querySelector('[name="ban"][type="checkbox"]')
+  permabanCheckbox = modalForm.querySelector('[name="permaban"][type="checkbox"]')
 
-  parent.find('#_ban').on "change", (event) ->
+  banCheckbox.addEventListener "change", (event) ->
     $t = $ this
     if $t.is(":checked")
       $("#ban-controls").show()
     else
       $("#ban-controls").hide()
-  parent.find('#_permaban').on "change", (event) ->
+  permabanCheckbox.addEventListener() "change", (event) ->
```

Comment:
```suggestion
  permabanCheckbox.addEventListener "change", (event) ->
```

Not even sure if this qualifies as a syntax error in CoffeeScript's book, but at least choose to keep it same-y. Removing the parenthesis would adjust it to the same style as all the other `addEventListener` calls you made.